Agent/Main/MainAgent.ts

```
@@ -21,6 +26,13 @@ export function sendWorkflow(ws: WebSocket, workflow: string, timeoutMs = 30000)
       clearTimeout(timeoutId);
     };
 
+    let timeoutId: ReturnType<typeof setTimeout>;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'timeoutId'. Consider to delete it or rename it.

'timeoutId' is defined here:


(lint/suspicious/noRedeclare)